src/Dockerfile.dev

```diff
@@ -0,0 +1,33 @@
+FROM php:7.4-fpm-alpine
+
+WORKDIR /app
+
+RUN apk add --no-cache \
+    gettext \
+    caddy \
+    nss-tools \
+    libzip-dev \
+    libpng-dev \
+    libjpeg-turbo-dev \
+    jpeg-dev \
+    icu-dev
+
+RUN docker-php-ext-configure gd --with-jpeg
+
+RUN docker-php-ext-install \
+    zip \
+    gd \
+    pdo_mysql\
+    intl \
+    opcache
+
+RUN echo 'memory_limit = 512M' >> /usr/local/etc/php/conf.d/docker-php-memlimit.ini && \
+    echo 'php_admin_flag[display_errors] = on' >> /usr/local/etc/php/conf.d/docker-php-logs.ini
```

Comment:
If this echo is the only difference between this image and the base one, maybe merge them? source: https://stackoverflow.com/a/53289193

Reply:
Sure, good idea.